### PR TITLE
fix: minor updates

### PR DESCRIPTION
**Pull request by henryiii**
Minor updates to the page.

### CODE REVIEW

File: pages/developers/style.md [hunk 1]
```diff
@@ -219,17 +219,33 @@ files = src
...
 There are a lot of options, and you can start with only typing global code and
 functions with at least one type annotation (the default) and enable more
-checks as you go. You can ignore missing imports on libraries as shown above,
-on section each. And you can disable MyPy on a line with `# type: ignore`. Once
-you are ready to start checking more, you can look at adding
-`check_untyped_defs`, `disallow_untyped_defs`, `disallow_incomplete_defs`, and
-more. You can add these *per file* by adding a `# mypy: <option>` the top of
+checks as you go (possibly by slowly uncommenting items in the list above).
+You can ignore missing imports on libraries as shown above,
+on section each. And you can disable MyPy on a line with `# type: ignore`.
```
Comment (eduardo-rodrigues) [inline pages/developers/style.md:245]: What do you mean by "on section each"? Words in the wrong order I guess.

File: pages/developers/style.md [hunk 2]
```diff
@@ -219,17 +219,33 @@ files = src
...
+One strategy would be to enable `check_untyped_defs` first, followed by
+`disallow_untyped_defs` then `disallow_incomplete_defs`.
+You can add these *per file* by adding a `# mypy: <option>` the top of
```
Comment (eduardo-rodrigues) [inline pages/developers/style.md:248]: "at the top of"?

**Review by eduardo-rodrigues — APPROVED** [review]
A couple of things worth checking. Otherwise great.

File: pages/developers/style.md (hunk 1 above)
Comment (henryiii) [reply pages/developers/style.md:245]: one section each.

**Comment by henryiii** [comment]
Thanks!

**Pull request merged**